README.md

```diff
@@ -1,22 +1,54 @@
-# kubescape.io website
+# kubescape.io Website
 
-This repository contains the source files for the kubescape.io website.
+This repository contains the source files for the [kubescape.io](https://kubescape.io) website. The website is built using [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme.
 
-## Building the site
+## Table of Contents
 
-### Local testing
+- [Getting Started](#getting-started)
+- [Local Testing](#local-testing)
+- [Production](#production)
+- [Contributing](#contributing)
+- [License](#license)
 
-We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme. 
+## Getting Started
 
-To build and test locally: 
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 
-* [install mkdocs-material](https://squidfunk.github.io/mkdocs-material/getting-started/) with `pip` or Docker 
-* run `mkdocs serve`
+### Prerequisites
 
-## Local build
+- Python installed on your local machine
+- pip 
 
-Run `mkdocs build`, and the site will be built in the `site/` directory.
+### Setting Up
 
-### Production
+1. Clone the repository to your local machine.
+```bash
+git clone https://github.com/kubescape/website.git
```

Comment:
The repo isn't called `website`, it's called `kubescape.io`

I don't think we need to go to the level of teaching people how to check out the website in the readme.  Assume a basic level of background.

Reply:
So would you suggest I remove the `setting up` section ? or Keep it short in just 2 points ?👇🏻

- Clone
- Install dependencies through requirements